scripts/nope.py

```diff
@@ -304,6 +304,23 @@ def dump_data_locally(timestamp, partial=False):
     # return if no issues
     return None
 
+def format_for_upload():
+    ''' formats the data in RESULTS into a de-normalized payload for elasticsearch
+        returns a map with the de-normalized array under the 'data' key
+    '''
```

Comment:
```suggestion
    ''' formats the data in RESULTS into a de-normalized payload for elasticsearch
        returns a dict with the de-normalized list under the 'data' key
    '''
```
JS/Python terms :wink: but if we go the approach I suggest below we could just do
```suggestion
    ''' formats the data in RESULTS into a de-normalized list for elasticsearch upload compatability
    '''
```